### PR TITLE
Break username when too long for navbar

### PR DESCRIPTION
**Pull request by sujeebant**

### CODE REVIEW

**Review by G-Ray — APPROVED** [review]

File: src/containers/Landing/Landing.module.scss [hunk 1]
```diff
@@ -13,8 +13,8 @@
   }
 
   .section1 {
-    padding-top: 68px;
-    height: 666px;
+    padding: 68px 20px 0 a20px;
```
Comment (G-Ray) [inline src/containers/Landing/Landing.module.scss]: typo ?

Comment (G-Ray) [reply src/containers/Landing/Landing.module.scss]: a20px

File: src/containers/Login/Login.module.scss [hunk 2]
```diff
@@ -43,6 +43,8 @@
     margin: auto;
     cursor: pointer;
     text-decoration: none;
+    max-width: 120px;
+    margin-top: 0;
```
Comment (G-Ray) [inline src/containers/Login/Login.module.scss]: I think you can remove `margin-top` and even `margin: auto` properties as they do not seem to have effect ?

File: src/containers/Landing/Landing.module.scss (hunk 1 above)
Comment (sujeebant) [reply src/containers/Landing/Landing.module.scss]: yes

File: src/containers/Login/Login.module.scss (hunk 2 above)
Comment (sujeebant) [reply src/containers/Login/Login.module.scss]: instead of adding max-width and margin-top, I just removed margin: auto which has the same effect

**Pull request merged**